### PR TITLE
Fix link to documentation

### PR DESCRIPTION
**Pull request by LeeDumond**
Fix link to point to current EF documentation

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[aspnetcore/data/ef-mvc/advanced.md](https://github.com/LeeDumond/Docs/blob/patch-14/aspnetcore/data/ef-mvc/advanced.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C26%5C6a0d29ae-bb37-09f1-c55a-4cfa862c6784%5CPullRequest%5C201611260440276942-2254%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: aspnetcore/data/ef-mvc/advanced.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ To reverse engineer a data model including entity classes from an existing datab
 
 ## Summary
 
-This completes this series of tutorials on using the Entity Framework Core in an ASP.NET MVC application. For more information about how to work with data using the Entity Framework, see the [EF documentation](https://ef.readthedocs.io/).
+This completes this series of tutorials on using the Entity Framework Core in an ASP.NET MVC application. For more information about how to work with data using the Entity Framework, see the [EF documentation](https://docs.microsoft.com/en-us/ef/).
```
Comment (tdykstra) [inline aspnetcore/data/ef-mvc/advanced.md]: Links should omit the locale node ("/en-us")

Comment (LeeDumond) [inline aspnetcore/data/ef-mvc/advanced.md]: True, I hadn't thought of that...

**Comment by Rick-Anderson** [comment]
I removed local - @LeeDumond  thanks

**Comment by Rick-Anderson** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[aspnetcore/data/ef-mvc/advanced.md](https://github.com/LeeDumond/Docs/blob/patch-14/aspnetcore/data/ef-mvc/advanced.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C11%5C28%5C6a0d29ae-bb37-09f1-c55a-4cfa862c6784%5CPullRequest%5C201611282155350678-2254%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**